content/docs/instrumenting/exporters.md

```diff
@@ -224,6 +224,7 @@ separate exporters are needed:
    * [App Connect Enterprise](https://github.com/ot4i/ace-docker)
    * [Ballerina](https://ballerina.io/)
    * [BFE](https://github.com/baidu/bfe)
+   * [BigBlueButton](https://github.com/greenstatic/bigbluebutton-exporter)
```

Comment:
This looks to be an exporter for an API, rather than an application itself

Reply:
> This looks to be an exporter for an API, rather than an application itself

Yes that is correct, the exporter uses the applications [official API](https://docs.bigbluebutton.org/dev/api.html) that is documented on their website for integrations.

Reply:
Yes, you've put it in the wrong section.

Reply:
My apologies, which section is appropriate?